docs/sources/installation/upgrading.md

```diff
@@ -347,12 +347,34 @@ Grafana v8.0 changes the underlying data structure to [data frames]({{< relref "
 For any existing panels/visualizations using a _Time series_ query, where the time column is only needed for filtering the time range, for example, using the bar gauge or pie chart panel, we recommend that you use a _Table query_ instead and exclude the time column as a field in the response.
 Refer to this [issue comment](https://github.com/grafana/grafana/issues/35534#issuecomment-861519658) for detailed instructions and workarounds.
 
+#### Prefix added to series names
+
+Given you have a query similar to below, where there's a time value and a numeric value selected together with a string value that's not named _metric_. Then you might notice that the graph panel renders series names as `value <hostname>` rather than just `<hostname>` which was the case before Grafana 8.
```

Comment:
```suggestion
When you have a query where there's a time value and a numeric value selected together with a string value that's not named _metric_, the graph panel renders series names as `value <hostname>` rather than just `<hostname>` which was the case before Grafana 8.
```